meiki_server/main.go

```diff
@@ -14,8 +15,16 @@ import (
 	"go.uber.org/zap"
 )
 
+func getEnv(key, defaultValue string) string {
```

Comment:
You had to write a function for this? #justgothings

Reply:
Yep